docs/main/101.md

```diff
@@ -1,6 +1,8 @@
 # ArmaForces 101
 
-# Przed dołączeniem
+# Przed dołączeniem - [Poradnik na 101](https://docs.google.com/document/d/1nOPqnTeu9Flph8oBtN2FFMQ5dUvwFet_Y80uH1ckp1s/edit?usp=sharing)
+
+Od deski do deski zapoznaj się z [Poradnikiem na 101 dla trzody i kaszy gryczanej](https://docs.google.com/document/d/1nOPqnTeu9Flph8oBtN2FFMQ5dUvwFet_Y80uH1ckp1s/edit?usp=sharing). Pokrywa on podstawowe zagadnienia pokroju zapisywania się na pierwszą misję i instalacji odpowiednich modów, z którymi każdy rekrut powinien się zapoznać. 
```

Comment:
I don't know if `trzoda chlewna i kasza gryczana` is the best welcome we can give. 🤣

Maybe we could create real homepage for wiki so this will not be the first thing a user sees but this is out of this PR scope.

Reply:
Toned it down a little bit ;), added proper pages for each of the new guides.